.github/workflows/cli-release.yaml

```diff
@@ -0,0 +1,25 @@
+on:
```

Comment:
Remove `.yaml` from the filename. Otherwise this `.yaml`. extension will be visible in the status badge, which looks not very nice.

Reply:
No. .yaml extension should be preserved